x/crosschain/client/integrationtests/inbound_voter_test.go

```diff
@@ -240,17 +240,23 @@ func (s *IntegrationTestSuite) TestCCTXInboundVoter() {
 			s.Require().NoError(s.network.WaitForNBlocks(2))
 
 			// Vote the tss
-			for _, val := range s.network.Validators {
-				out, err := clitestutil.ExecTestCLICmd(broadcaster.ClientCtx, authcli.GetAccountCmd(), []string{val.Address.String(), "--output", "json"})
-				s.Require().NoError(err)
-
-				var account authtypes.AccountI
-				s.NoError(val.ClientCtx.Codec.UnmarshalInterfaceJSON(out.Bytes(), &account))
-				signedTx := BuildSignedTssVote(s.T(), val, s.cfg.BondDenom, account)
-				out, err = clitestutil.ExecTestCLICmd(broadcaster.ClientCtx, authcli.GetBroadcastCommand(), []string{signedTx.Name(), "--broadcast-mode", "sync"})
-				s.Require().NoError(err)
-			}
+			//for _, val := range s.network.Validators {
```

Comment:
What is the context for commenting this out?
Can we remove it if no longer needed?

Reply:
745da8ca35cb6e1d6679310c1f979aa9d1ae1eb4